multichannel_timeseries/anaconda-project.yml

```diff
@@ -0,0 +1,79 @@
+name: multichannel_timeseries
+description: Set of examples to demonstrate the display and annotation of multichannel timeseries data in Neuroscience with HoloViz libraries.
+
+examples_config:
+  created: 2024-08-28
+  maintainers:
+  - "droumis"
+  labels:
+  - "hvplot"
+  - "panel"
+  - "holoviews"
+  categories:
+  - Neuroscience
+  title: "Multichannel Timeseries"
+  deployments:
+  - command: notebook
+  - command: dashboard
+    resource_profile: default
+    auto_deploy: true
+  notebooks_to_skip: []
+  skip_notebooks_evaluation: false
+  no_data_ingestion: false
+  gh_runner: "ubuntu-latest"
+
+user_fields: [examples_config]
+
+channels:
+- conda-forge
+- nodefaults
+
+packages: &pkgs
+- notebook>=6.5.2,<7 # required
+- python=3.11
+- numpy
+- panel>=1.4.2
+- hvplot>=0.10.0
+- pandas>=2.2.1
+- holoviews>=1.19.0 # 1.19: wide df handling, scalebars
+- datashader
+- xarray>=2024.5.0
+- zarr
+- pyarrow
+- dask
+- h5py
+- mne-lsl
+- jupyterlab
+- pip
+- pip:
+  - holonote>=0.2.1 # 0.2.1: visibility and coloring fixes
```

Comment:
ok. Just noting in case it pops up in the future.. I have inconsistently seen weird behavior with mne when installed via conda.. sometimes things break and the `.io` module is missing.

Reply:
can confirm, installing mne via conda will sometimes break things. I'm reverting to installing it via pip

Reply:
Ok weird but that's how it is! Could you leave a comment in the project file stating why `mne` is in the `pip` section then?